redis/redis_test.go

```diff
@@ -33,52 +34,52 @@ var _ = Describe("Redis client", func() {
 
 		DeferCleanup(redisServer.Close)
 
-		var rcfg config.RedisConfig
+		var rcfg config.Redis
 		err = defaults.Set(&rcfg)
 
 		Expect(err).Should(Succeed())
 
 		rcfg.Address = redisServer.Addr()
 		redisConfig = &rcfg
-		redisClient, err = New(redisConfig)
+		redisClient, err = New(context.TODO(), redisConfig)
```

Comment:
You could use the same technique I used for resolvers:
```go
	var (
		ctx       context.Context
		cancelFn  context.CancelFunc
	)

	BeforeEach(func() {
		ctx, cancelFn = context.WithCancel(context.Background())
		DeferCleanup(cancelFn)
	})
```

Or ideally we'd use ginkgo's support for having a context per test, but that only works when calling`New` from a specific test and not in setup nodes ([docs](https://onsi.github.io/ginkgo/#interruptible-node-function-signatures-a-quick-reference)):
```go
It("is interruptible", func (ctx context.Context) {
  ...
})
```

Reply:
Missed the part of the docs, I'll read into em.

Reply:
According to the documentation setup nodes can also use contexts:

"Most Ginkgo nodes can be made interruptible. Setup and Subject nodes typically take a simple func() {} but can be made interruptible like so:"